dist/ng-obiba-mica.js

```diff
@@ -3022,7 +3022,7 @@ angular.module('obiba.mica.search')
             target: target,
             taxonomy: taxonomy
           }).$promise.then(function (taxonomy) {
-            vocabulary = taxonomy.vocabularies.filter(function (v) {return v.name === vocabulary; })[0];
+            vocabulary = taxonomy.vocabularies.filter(function (v) {return v.name === vocabulary || RqlQueryUtils.vocabularyAlias(v) === vocabulary; })[0];
```

Comment:
It'll do the same as before, we will have a type error in javascript and we won't be redirected.
I'm not familiar enough with the criteria item creation to give an alternative.
We could catch the error and log it but if it can't create the item, it won't do so.